cmd/agent/subcommands/status/command.go

```diff
@@ -96,8 +96,10 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 			)
 		},
 	}
+	componentCmd.Flags().BoolVarP(&cliParams.jsonStatus, "json", "j", false, "print out raw json")
```

Comment:
**❓ question:** ‏I think we might want to make these persistent flags (see [example](https://github.com/DataDog/datadog-agent/blob/42997a45690e4f57d198ef2a0114f51c2391fe3c/cmd/agent/command/command.go#L73-L74)) but I'm not sure if it's technically possible with cobra to make these global config only for the `status` subcommand (e.g. it should not impact other subcommands)

Reply:
Good point, I replaced it !